docs/run-a-node/setup/InstallANode.md

```diff
@@ -0,0 +1,72 @@
+title: Install a node
+
+### Overview
+
+This guide explains how to install the Algorand Node software on Linux distributions and Mac OS. 
+
+When installing on Debian based distribution or Red Hat based distribution the package manager configures the systemd algod daemon service. The package manager method uses fixed directories and automatically updates. It has been validated on Debian, Ubuntu, Fedora, and CentOS.
+
+When installing on Other Linux or Mac the updater script method will download and requires manually setting data directories and requires manual updates. It has been tested on the same Linux distributions from above, as well as on openSUSE Leap, Manjaro, Mageia, Alpine, and Solus.
+
+Read the hardware requirements below to confirm specs:
+
+- [Hardware requirements](#hardware-requirements)
+
+Then install using one of the following:
+
+- [AlgorandDocumentation.InstallANodeOnDebianBasedDistribution](Install a Node on Debian based distributions)
+- [AlgorandDocumentation.InstallANodeOnRedHatBasedDistribution](Install a Node on Red Hat based distributions)
+- [AlgorandDocumentation.InstallANodeOnOtherLinux](Install a Node on Other Linux)
+- [AlgorandDocumentation.InstallANodeOnMac](Install a Node on a Mac)
+
+After installation is complete, the following configurations are available for customization:
+
+- [AlgorandDocumentation.GivePermissionsRunAsAlgorandUser](Give user permissions to run as algorand user)
+- [AlgorandDocumentation.InstallANodeConfigureAlgodService](Configure algod as a systemd service)
+- [AlgorandDocumentation.InstallANodeConfigureNodeTelemetry](Configure node to use Telemetry)
+- [AlgorandDocumentation.InstallANodeSyncNodeFastCatchup](Sync node using Fast Catchup)
+- [AlgorandDocumentation.InstallANodeSwitchNetworks](Switching Networks - BetaNet, TestNet, MainNet)
+

```

Comment:
Same with these - I think they're not linking to the right place/formatted correctly.